commercetools/commercetools-sdk-java-api/src/integrationTest/java/commercetools/me/MyCartsFixtures.java

```diff
@@ -0,0 +1,70 @@
+package commercetools.me;
+
+import com.commercetools.api.models.me.MyCart;
+import com.commercetools.api.models.me.MyCartDraft;
+import com.commercetools.api.models.me.MyCartDraftBuilder;
+import commercetools.utils.CommercetoolsTestUtils;
+import org.junit.Assert;
+
+import java.util.function.Consumer;
+import java.util.function.UnaryOperator;
+
+import static commercetools.cart.CartsFixtures.deleteCart;
+
+public class MyCartsFixtures {
+    public static MyCart deleteMyCart(final String id, final Long version) {
+        MyCart myCart = CommercetoolsTestUtils.getProjectRoot()
```

Comment:
these methods will not work as the client in TestUtils is most likely not properly configured to be able to use the Me endpoints. So either change them to accept a ProjectRoot or delete them